Enterprise-CMCS/managed-care-review
# [MCR-3547] UI regression fix for MCCRS ID and State Contacts

## Summary
This is a small fix for the displayed MC-CRS ID being out of alignment 
<img width="696" alt="Screenshot 2023-11-08 at 12 26 26 PM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/cfc0ebab-8704-4fb1-8093-c2f22d4089c3">

And for excess padding on the state contacts section of the submission summary
<img width="661" alt="Screenshot 2023-11-08 at 2 39 43 PM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/65009fa9-0bf5-41a6-81cf-15dc72ffe251">



#### Related issues
https://qmacbis.atlassian.net/browse/MCR-3547

#### Screenshots
<img width="1042" alt="Screenshot 2023-11-08 at 5 19 04 PM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/674d4212-b61a-4cf6-b103-b6a00b30d635">


<img width="926" alt="Screenshot 2023-11-09 at 11 36 05 AM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/422925c5-9039-40eb-b05c-0eed1829ccbe">

#### QA Steps
- Login as a CMS user
- Add a MCCRS ID to a submission
- When redirected to the submission summary page, the MC-CRS ID should be styled correctly 
- Scroll to the state contact section of the submission summary and ensure the UI is correct
